src/hemanshi/Test4.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Test 4 - 11th Sep'2022
+Expected time : 15 mins
+Program 1 :
+Write a method to return sum of all the digits present in each elements from given array.
+input : {"Ma1he4sh","A9vni","P1uj3a5","Harshal","A4d4itty","Rahul"}
+output : Mahesh -> 5
+         Avni -> 9
+         Puja -> 9
+         Harshal -> 0
+         Aditty -> 8
+         Rahul -> 0
+Program 2 : Return count of visible trees. [15 - 20 mins, max - 30 mins]
+input : {3,7,5}
+output : 2
+input : {3,5,11,9,10,13}
+output : 4
+ */
+
+package hemanshi;
+
+public class Test4 {
+	
+//program-1
+	int getDigitSum(String name) {
+		int sum = 0;
+		for(int j = 0; j < name.length(); j++) {
+			if(Character.isDigit(name.charAt(j))) {
+				int num = Character.getNumericValue(name.charAt(j));
+				sum += num;
+			}
+		}
+		return sum;
+	}
+	void getSumOfDigitInString(String[] name) {
+		int count = 0;
+		for(int index = 0; index < name.length; index++) {
+			String currentName = name[index];
+			System.out.println(currentName + "---" + getDigitSum(currentName));
+			count++;
+			}
+		}
+		
+//program-2
+	int checkTreesVisible(int[] treesHight){
+		int counttreeVisible = 0;
```

Comment:
counttreeVisible should be started from 1, as the first tree always visible in the row